tests/models/test_internvl.py

```diff
@@ -64,13 +65,13 @@ def generate(
 def run_test(
     hf_runner: Type[HfRunner],
     vllm_runner: Type[VllmRunner],
-    image_assets: _ImageAssets,
+    inputs: List[Tuple[List[str], Union[List[Image], List[List[Image]]]]],
```

Comment:
Minor nit: You can import `PromptImageInput` from `conftest.py`.

Reply:
OK, I also updated this in `test_phi3v.py` by the way.